src/amo/components/ScreenShots.js

```diff
@@ -37,14 +35,14 @@ const formatPreviews = (previews) => (
   previews.map((preview) => ({
     src: preview.image_url,
     thumbnail_src: preview.thumbnail_url,
-    h: HEIGHT,
-    w: WIDTH,
+    h: preview.image_size[1],
+    w: preview.image_size[0],
     title: preview.caption,
   }))
 );
 
 export const thumbnailContent = (item) => (
-  <img src={item.src} className="ScreenShots-image" height={HEIGHT} width={WIDTH} alt="" title={item.title} />
+  <img src={item.src} className="ScreenShots-image" height={item.h} width={item.w} alt="" title={item.title} />
```

Comment:
We could probably use `item.title` as `alt` content. It is not a super-best-practice, but it is better than having an empty `alt` I guess.

Reply:
For accessibility, we should not have the same content for both `alt` and `title`.

`title` should be the title of the image (e.g. [The Treachery of Images](https://en.wikipedia.org/wiki/The_Treachery_of_Images)) and `alt` should be a description of what the image shows (e.g., `The picture shows a pipe. Below it, Magritte painted, "Ceci n'est pas une pipe.", French for "This is not a pipe."`).